kahuna/public/js/components/gr-image-usage/gr-image-usage.html

```diff
@@ -0,0 +1,35 @@
+<div ng:show="ctrl.usage.pending || ctrl.usage.published">
```

Comment:
Just fiddling in this branch. Agreed UI work should be in another branch here, but the project split makes them effectively releasable separately. I think @akash1810 is working on UI stuff at the moment, maybe we could cherry pick these commits into another branch and remove them here.